src/methods/SANA.cpp

```diff
@@ -2290,9 +2289,14 @@ void SANA::trackProgress(long long int i, bool end) {
     bool printScores = false;
     bool checkScores = true;
     double fractional_time = i/(double)_maxExecutionIterations;
-
+    
     cout << i/iterationsPerStep << " (" << 100*fractional_time << "%," << timer.elapsed() << "s): score = " << currentScore;
     cout <<  " P(" << avgEnergyInc << ", " << Temperature << ") = " << acceptingProbability(avgEnergyInc, Temperature) << ", pBad = " << trueAcceptingProbability() << endl;
+    
+    double elapsedTime = timer.elapsed();
+    if(elapsedTime != 0)
+        cout << "SANA is currently doing " << to_string(1E6/(elapsedTime-oldTimeElapsed)) << " iterations per second" << endl;
```

Comment:
Uh... why is 1e6 in the numerator? It should be the actual number of iterations since the last check.